### PR TITLE
Feature/momza 1029 wa default reg

### PR DESCRIPTION
**Pull request by OMosimege**
This PR is implementing the whatsapp default registration for the CHW USSD line

### CODE REVIEW

File: go-app-ussd_chw.js [hunk 1]
```diff
@@ -379,23 +379,66 @@ go.app = function() {
             var registration_info = {
                 "reg_type": "momconnect_prebirth",
                 "registrant_id": self.im.user.answers.registrant.id,
+                "registration_type" : self.im.user.answers.registration_type,
```
Comment (rudigiesler) [inline go-app-ussd_chw.js]: This isn't how we set the registration type on SEED. It's set on the `reg_type` field as either `momconnect_prebirth` or `whatsapp_prebirth`

File: go-app-ussd_chw.js [hunk 2]
```diff
@@ -778,29 +821,65 @@ go.app = function() {
...
-                next: "state_save_subscription"
+                next: "state_set_registration_type"
+            });
+        });
+
+        self.add("state_set_registration_type", function(name){
+            /*var registration_types = {
+                'sms': 'SMS',
+                'whatsapp': 'WhatsApp',
+            };*/
```
Comment (rudigiesler) [inline go-app-ussd_chw.js]: We can probably remove this commented out code

File: go-app-ussd_chw.js (hunk 1 above)
Comment (OMosimege) [reply go-app-ussd_chw.js]: Is there a way to differentiate a whatsapp subscription to a non-whatsapp one?

File: go-app-ussd_chw.js (hunk 2 above)
Comment (OMosimege) [reply go-app-ussd_chw.js]: Thanks for pointing this out

File: go-app-ussd_chw.js (hunk 1 above)
Comment (rudigiesler) [reply go-app-ussd_chw.js]: Yes, the `reg_type` field determines what kind of registration it is.

For the `momconnect_prebirth`, it's an SMS registration, for `whatsapp_prebirth`, is a whatsapp registration

**Review by OMosimege — COMMENTED** [review]
👍

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**